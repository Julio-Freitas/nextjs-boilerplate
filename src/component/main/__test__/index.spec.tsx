import { render, screen } from '@testing-library/react';
import { Main } from '..';

describe('<Main />', () => {
  it('Should be render corretcly', () => {
    const { container } = render(<Main />);
    expect(screen.getByRole('heading')).toHaveTextContent('OlÃ¡ Mundo...');
    expect(container.firstChild).toMatchSnapshot();
  });

  it('Should be render corretcly', () => {
    render(<Main />);
    expect(screen.getByRole('heading')).toHaveStyle({ color: '#001523' });
  });
});
