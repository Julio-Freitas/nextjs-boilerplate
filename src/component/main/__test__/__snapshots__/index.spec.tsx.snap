// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Should be render corretcly 1`] = `
.c0 {
  color: #001523;
}

<h1
  class="c0"
>
  Ol√° Mundo...
</h1>
`;
